src/purview.ts

```diff
@@ -43,12 +43,25 @@ import {
   generateRule,
   getAtomicProperties,
 } from "./css"
+import { JSXInternal } from "./types/jsx"
 
+const InternalComponent = Component
```

Comment:
This reassignment is made necessary below when include the abstract class `Component` in the Purview namespace in order to preserve the API usage `Purview.Component<...> {...}`